client/src/data/Search

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useContext, useEffect} from "react";
```

Comment:
please delete this file....remember not to commit any files you don't want the rest of us to have.

Reply:
Thx! I thought I deleted it but it didn't...